### ISSUE TITLE
Webpack 5 support?

### ISSUE THREAD

**Issue by Joaoviana** [piglovesyou/graphql-let#526]
Hello 👋 

Was wondering if you were thinking of adding WP5 support to `graphql-let`!

I'm working on a NextJS project (v11) with WP5 and heavily use your package.

**Comment by sachinraja** [comment]
Works on Next.js with Webpack 5 for me. Check out [the example](https://github.com/vercel/next.js/tree/master/examples/with-typescript-graphql).

**Comment by piglovesyou** [comment]
@Joaoviana Good to know you use graphql-let! Would you mind listing what problems you find with it?

**Comment by Joaoviana** [comment]
Apologies for the late reply @piglovesyou , and @sachinraja!

This package works 100% OK with Next.js and Webpack 5 🎉 

For context, the issue I encountered was with importing queries, documents, and generated apollo hooks with from an index file that would import other `.graphql` file. 

For example:

`graphql/index.ts`
looks like the following:
```ts
export * from './my-queries.graphql
```

and 
`my-queries.graphql`
```
query {
 something
}
```
`graphq-let` would generate the following: `my-queries.graphql.d.ts`

after upgrading `graphql-let` to latest - and also using WP5 - I found out that I couldn't do the following in the app anymore: 
```ts
import {MyQueryDocument} from 'graphql/queries'
```

I would have to change it to: 
```ts
import {MyQueryDocument} from 'graphql/queries/my-query.graphql
```

This worked for me!

Feel free to close the issue 😄
